### DISCUSSION TITLE
直播回放列表为0

### DISCUSSION THREAD

**Discussion by Hiiro570** [HMBSbige/BilibiliLiveRecordDownLoader#133] [Q&A] [answered]
直播回放里面是空的，是有什么条件比如需要向服务器管理员申请之类的才会保存吗

**Comment by HMBSbige** [comment]
只能说明该主播没有直播回放

**Reply by HMBSbige** [reply]
不过好像最近很多主播的直播回放都没了，不知道什么原因

**Reply by Hiiro570** [reply]
可能是因为新版的直播回放的位置换掉了，现在的位置在（主播个人空间-频道-直播回放）这里面

**Comment by HMBSbige** [comment, marked as answer]
[B站公告](https://link.bilibili.com/p/eden/news#/newsdetail?id=2103)

看来改版完成后下载回放功能可以去掉了

**Reply by Hiiro570** [reply]
啊...辛苦了
